lib/acts_as_taggable_on/tag.rb

```diff
@@ -89,6 +89,30 @@ def self.find_or_create_all_with_like_by_name(*list)
       end
     end
 
+    def self.update_tag(tag_name, id)
+      tag = ActsAsTaggableOn::Tag.find_by(id: id)
+      if tag.present?
+        tag.name = tag_name
+        tag.save
+      end
+    end
+
+    def self.disable_tag(id)
```

Comment:
@usama-ashraf do we actually need those functions?

Can we just do something like `Tag.find(1).enable`

Reply:
@usama-ashraf or you can at least call it `disable` and `enable`, so we can use it like

`Tag.disable(2)` using `_tag` in the name is redondant.

Reply:
changes done @fbrunel